### ISSUE TITLE
Tariff structure processing (to human-readable text)

### ISSUE THREAD

**Issue by simonschllng** [ocpi/ocpi#108]
I'm wondering if we could describe an algorithm that translates the tariff structure into human-readable text in multiple languages. This could be done together in the project. I've already started something, but I think more eyes on it would not harm.

The algorithm (in pseudo-code) could then be used by all participants to display the tariffs nicely, after adding own sales margins, etc.

**Comment by RobertdeLeeuw-IHomer** [comment]
I don't think that a generic solution is possible. It all depends on how complex a tariff is and how much characters can be used to display the text...
